libs/datasets/latest_values_dataset.py

```diff
@@ -185,6 +187,8 @@ def to_csv(self, path: pathlib.Path):
         """
         # Cannot use common_df.write_csv as it doesn't support data without a date index field.
         data = self.data.set_index(CommonFields.FIPS).replace({pd.NA: np.nan}).convert_dtypes()
+        data = common_df.only_common_columns(data, structlog.get_logger())  # Drops `index`
+        data = common_df.sort_common_field_columns(data).sort_index()
```

Comment:
Should we move this into the common_df? The main reason it's not there is because `common_df` mostly expects a date column, but a good portion of datasets won't have that.

Reply:
I think the bad portion of datasets don't have a date column :-P Medium term I'd like to stop using latest.csv (to reduce the number of different indexes we use, so adding provenance is easier) so don't think it is worth shuffling the code around.